bin/aurbuild

```diff
@@ -114,6 +114,11 @@ else
     mkpkg_args+=("${@:$OPTIND}")
 fi
 
+if [[ $sign == --sign ]]; then
+    # Pass socket and environment to the build chroot.
+    mkchr_args+=("-d" "$HOME/.gnupg/S.gpg-agent:/build/.gnupg/S.gpg-agent")
+    mkchr_args+=("-e" "GPG_TTY=$(tty)" "-e" "DISPLAY=$DISPLAY")
```

Comment:
Then it passes an empty variable into the chroot and gpg looks for GPG_TTY instead — I thought about and tested that.